lisa/operating_system.py

```diff
@@ -618,6 +618,12 @@ def name_pattern(cls) -> Pattern[str]:
         return re.compile("^coreos|Flatcar|flatcar$")
 
 
+class Alpine(Linux):
+    @classmethod
+    def name_pattern(cls) -> Pattern[str]:
+        return re.compile("^Alpine")
```

Comment:
do you know it is centos or Debian type?

Reply:
It is not either type. It is a security-oriented, lightweight Linux distribution. It is designed for routers, firewalls, and virtual private networks. We have publishers using this system.